packages/manager/modules/vrack/src/partials/available.html

```diff
@@ -1,89 +1,138 @@
-<h2 class="vrack__section-title" data-translate="vrack_your_services"></h2>
-<ul class="vrack__service-type-list">
-    <li class="vrack__section-type" data-ng-if="$ctrl.loaders.deleting">
-        <div class="text-center">
-            <oui-spinner></oui-spinner>
-        </div>
-    </li>
-    <li
-        class="vrack__service-type"
-        data-ng-init="$ctrl.setAccordionState('available', 'types', $index, true)"
-        data-ng-repeat="(serviceType, services) in $ctrl.data.allowedServices track by serviceType"
-        data-ng-if="$ctrl.constructor.hasServices(services) && !$ctrl.loaders.deleting"
-        data-ng-class="{ 'vrack__service-type_expanded': $ctrl.getAccordionState('available', 'types', $index) }"
-    >
-        <span
-            class="vrack__service-type-name"
-            data-ng-click="$ctrl.toggleAccordion('available', 'types', $index)"
+<div class="vrack__available">
+    <h2 class="vrack__section-title">
+        <span data-translate="vrack_your_services"></span>
+    </h2>
+    <ul class="vrack__service-type-list">
+        <li
+            class="vrack__service-fetching pull-right"
+            data-ng-if="$ctrl.getAllowedServicesCounter < $ctrl.SERVICE_FAMILIES_LIST.length"
         >
-            <i
-                class="vrack__service-type-icon"
-                data-ng-class="{
-                'ovh-font ovh-font-dedicatedCloud': serviceType === 'dedicatedCloud',
-                'oui-icon oui-icon-cloud-essential_concept': serviceType === 'managedBareMetal',
-                'ovh-font ovh-font-public-cloud': serviceType === 'cloudProject',
-                'ovh-font ovh-font-iplb' : serviceType === 'ipLoadbalancing',
-                'ovh-font ovh-font-ip': serviceType === 'ip',
-                'ovh-font ovh-font-server': serviceType === 'dedicatedServer' || serviceType === 'dedicatedServerInterface',
-                'ovh-font ovh-font-network': serviceType === 'dedicatedConnect',
-                'ovh-font ovh-font-vRack': serviceType === 'legacyVrack'}"
+            <oui-spinner data-size="s"></oui-spinner>
+            <span
+                class="oui-badge oui-badge_info"
+                data-ng-bind="{{ $ctrl.getAllowedServicesCounter + '/' + $ctrl.SERVICE_FAMILIES_LIST.length }}"
+            ></span>
+        </li>
+        <li class="vrack__section-type">
+            <span
+                class="loading"
```

Comment:
You are sure this class (loading) exist ?

Reply:
it is not, i removed this

Reply:
removed